src/reference-manual/transforms.qmd

```diff
@@ -330,7 +330,7 @@ $$
 
 The default value for the offset $\mu$ is $0$ and for the multiplier $\sigma$ is
 $1$ in case not both are specified.
-
+For a container variable, the affine transform is applied to each element of that variable.
```

Comment:
I think there should be a space to start a new paragraph even though it's only one line.